wpeverest/everest-forms
# Enhancement - Set date range x days after or before.

### All Submissions:






### Changes proposed in this Pull Request:
This PR introduces "a date range x days after or before"  inside  "Enable Min Max" feature of the Date / Time Field.

### How to test the changes in this Pull Request:

1. Go to the Advanced Options of Date / Time field.
2. Check "Enable Min Max date" option then check "Enable Custom Input" option.
3. Enter the value in the 'Minimum Date' and 'Maximum Date' input fields.

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Set date range x days after or before.
